salt/modules/dockerng.py

```diff
@@ -416,6 +416,9 @@
     'cpuset': {
         'path': 'Config:Cpuset',
     },
+    'labels': {
+      'path': 'Config.Labels',
```

Comment:
Is ``Labels`` a sub-key of ``Config``? If so, then this should be ``Config:Labels`` (replace the dot with a colon).